packages/web-components/src/popover/popover.template.ts

```diff
@@ -0,0 +1,15 @@
+import { html, ref, slotted, when } from '@microsoft/fast-element';
+import { Popover } from './popover';
+
+export const popoverTemplate = () => html<Popover>`
+  <slot name="anchor" ${slotted('anchorRef')}></slot>
```

Comment:
This is an interesting approach to couple these together. Is there a scenario where we might not want to tie this to a single slotted element per popover? IE, would we potentially want to associate one popup instance across various id's on the page?